qdrant_client/embed/schema_parser.py

```diff
@@ -121,7 +121,7 @@ def _replace_refs(
                     schemes[k] = self._replace_refs(schema=v, parent=k, seen_refs=copy(seen_refs))
                 else:
                     schemes[k] = self._replace_refs(
-                        schema=v, parent=parent, seen_refs=copy(seen_refs)
```

Comment:
@joein applying this change fixes the infinite recursion

Reply:
Thanks a lot for looking into this, but...

Am I not looking right? This will make both branches of `if..else..` exactly the same

Reply:
Yes it does make both branch the same, that's why I'd like @joein to review the initial intent here because all tests are green otherwise.